en/docs/develop/configure-my-account.md

```diff
@@ -0,0 +1,404 @@
+
+## Configure session information
+
+Parameters such as how often the state of the session should be checked, how long a user should be allowed to remain idle etc., can be configured using the `session`. The `session` information has the following attributes.
+
+|Attribute|Description|Default Value|
+|--|--|--|
+| `"userIdleTimeout"`|This specifies the number of seconds a user is allowed to remain idle after which they will be logged out automatically. | 600|
+|`"userIdleWarningTimeout"`| This specifies how long, in seconds, a user needs to be idle before a warning notification is displayed.| 580|
+|`"sessionRefreshTimeout"`| This specifies how often the session should be refreshed. | 300|
+|`"checkSessionInterval"` | This specifies how often the session state should be checked. |3|
+
+## Disable using authenticator App as an authentication factor
+
+WSO2 Identity Server allows multi-Factor authentication via SMS, FIDO, and Authenticator Applications. The use of authenticator app can be disabled by setting the `enabled` key of the `authenticatorApp` under `ui` to `false`.
+
+```json
+"ui":{
+	"authenticatorApp":{
+		"enabled": false
+	}
+}
+```
+
+## Enable and Disable Application Features
+
+**My Account** has several features and those features, in turn, have several sub features. These features can be configured by using the `"features"` attribute under `"ui"`. Each feature under the `"features"` attribute has the following format.
+```json
+"feature":{
+	"disabledFeatures": [],
+	"enabled": true,
+	"scopes": {
+		"create": [],
+		"read": [],
+		"update": [],
+		"delete": []
+	}
+}
+```
+The `"disabledFeatures"` attribute is used to disable sub features belonging to a feature.
+
+The `"enabled"` attribute can be used to toggle the feature as a whole. For instance, by setting the `"enabled"` attribute of the `"overview"` feature, you can disable the `"overview"` feature as a whole.
+
+The `"scopes"` attribute is used to specify the scopes that a user should be allowed in order to access the concerned feature. The `"scopes"` attribute has the following sub-attributes and each sub-attribute takes an array of scopes.
+```json
+"scopes":{
+	"create":[],
+	"read":[],
+	"update":[],
+	"delete":[]
+}
+```
+If you want a user to have a certain scope(s) to be able to create an application, then you can specify that/those scope(s) within the `"create"` array of the application feature.
+
+### The features
+
+These are the five features in **My Account**.
+
+1.  Overview
+2.  Personal Info
+3.  Security
+4.  Applications
+5.  Operations
+
+#### 1. Overview
+
+The overview feature can be configured via the `"overview"` attribute of the `"features"` attribute under `"ui"`.
+
+```json
+"ui":{
+	"features":{
+		"overview":{
+			"enabled": true,
+			"disabledFeatures": [],
+			"scopes":{
+				"create": [],
+				"read": [],
+				"updated": [],
+				"delete": []
+			}
+		}
+	}
+}
+```
+Set the `"enabled"` attribute to `false` to disable Overview altogether.
```

Comment:
Overview --> should be overview or should be bolded ?

Reply:
please check wherever applicable for other terms as well like "Personal Info" below